pages/version-policy/index.md

```diff
@@ -69,14 +69,8 @@ The following matrix shows the platform components and operating environments on
     </tr>
     <tr>
         <td>CentOS 7.4</td>
-        <td><p style="text-align: center;"><ul><li>RH Fork of Docker CE 1.13.1<sup>*</sup></li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>RH Fork of Docker CE 1.13.1<sup>*</sup></li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>RH Fork of Docker CE 1.13.1<sup>*</sup></li></ul></p></td>
-    </tr>
-    <tr>
-        <td>CentOS 7.3</td>
-        <td><p style="text-align: center;"><ul><li>RH Fork of Docker CE 1.13.1<sup>*</sup></li></ul></p></td>
-        <td><p style="text-align: center;"><ul><li>RH Fork of Docker CE 1.13.1<sup>*</sup></li></ul></p></td>
+        <td><p style="text-align: center;"><ul><li>Docker 18.09.2 CE</li></ul></p></td>
```

Comment:
Lines 72 & 73, can you have it say `Docker CE 18.09.2` instead of `Docker 18.09.2 CE`? This is for CentOS 7.4.